__tests__/notifications/index.ts

```diff
@@ -326,6 +329,90 @@ describe('generateNotification', () => {
     ]);
   });
 
+  it('should update notification attachment when post type changes', async () => {
```

Comment:
We have triggers test on a separate file if not mistaken

Reply:
Sorted 478b4a0180c97ba39240250aeaca676ba8ab00c8